### PR TITLE
[hotfix] Remove argument BUILID_ID from run.sh

### PR DESCRIPTION
**Pull request by dmsolr**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

Having to configuration BUILD_ID and depending on Jenkinsfile's NO. is absurd.

### CODE REVIEW

**Comment by wu-sheng** [comment]
@dmsolr There are many Jenkins files

File: Jenkinsfile-Agent-Test [hunk 1]
```diff
@@ -63,12 +64,12 @@ pipeline {
                 }
             }
             steps {
-                sh './mvnw -f test/plugin/pom.xml clean package -DskipTests -Dbuild_id=wl1_${BUILD_ID} docker:build'
+                sh './mvnw -f test/plugin/pom.xml clean package -DskipTests -Dbuild_id=${BUILD_NO} docker:build'
```
Comment (kezhenxu94) [inline Jenkinsfile-Agent-Test]: I don't think we need `--build_id=wl1_${BUILD_ID}` any more

File: test/plugin/run.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ home="$(cd "$(dirname $0)"; pwd)"
 scenario_name=""
 parallel_run_size=1
 force_build="off"
-build_id="local"
+build_id="${BUILD_NO:=local}"
```
Comment (kezhenxu94) [inline test/plugin/run.sh]: Since you have use the environment variable here

**Comment by wu-sheng** [comment]
@kezhenxu94 Recheck please, if it is OK, let's still merge this into 6.5.0

**Pull request merged**